packages/component/src/BasicSendBox.js

```diff
@@ -9,10 +10,6 @@ import SendButton from './SendBox/SendButton';
 import SuggestedActions from './SendBox/SuggestedActions';
 import TextBox from './SendBox/TextBox';
 import UploadButton from './SendBox/UploadButton';
-import useActivities from './hooks/useActivities';
-import useDirection from './hooks/useDirection';
-import useDictateState from './hooks/useDictateState';
-import useStyleOptions from './hooks/useStyleOptions';
 import useStyleSet from './hooks/useStyleSet';
 import useStyleToEmotionObject from './hooks/internal/useStyleToEmotionObject';
 import useWebSpeechPonyfill from './hooks/useWebSpeechPonyfill';
```

Comment:
expand from `hooks`?

Reply:
The hooks are inside the same package, so we are not importing from `hook` then expanding it again. But good call.